docs/devWithDocker.md

```
@@ -8,34 +8,28 @@ Follow [the docker for desktop](https://www.docker.com/products/docker-desktop/)
+The `.env.docker` file is required to contain some environment variables that are not yet available through the configuration. [It's located at the root of the `transition` repo](../.env.docker).
 
-Keep this access token for the next step.
+The default configuration uses OpenStreetMap tiles which don't require any API keys. If you want to use custom raster tiles (such as aerial imagery), you can configure the `CUSTOM_RASTER_TILES_XYZ_URL` variable in the `.env.docker` file:
 
-
-### 3. Add your mapbox token to the .env.docker file
-
-The `.env.docker` file is required to contain some environment variables that are not yet available through the configuration. The file has already been created for you, you simply need to finish the configuration. [It's located at the root of the `transition` repo](../.env.docker).
-
-In order for the map to be displayed correctly, simply modify the field `MAPBOX_ACCESS_TOKEN` with the key you've acquired in the previous section. You must not push your key to the remote repo.
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)